app/controllers/v0/benefits_claims_controller.rb

```diff
@@ -5,21 +5,30 @@
 module V0
   class BenefitsClaimsController < ApplicationController
     def index
-      claims = service.get_claims
+      claims = service.get_claims(settings.access_token.client_id, settings.access_token.rsa_key)
```

Comment:
We removed these arguments on get_claims and get_claim on Friday - so we can avoid changes to Claim Status Tool Team's code.

Reply:
Ah thanks for noting. I think some of the diffs here are because this branch is downstream from yours; if we removed it, I'll either update with yours, or master after yours merges in